src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -159,25 +160,27 @@ public IEnumerable<IndividualPKAnalysis> CalculateFor(IReadOnlyList<Simulation>
 
       private IEnumerable<DataColumn> columnsFor(CurveData<TimeProfileXValue, TimeProfileYValue> curveData, IPopulationDataCollector populationDataCollector)
       {
-         var baseGrid = new BaseGrid(Constants.TIME, curveData.XAxis.Dimension) {Values = curveData.XValues.Select(x => x.X).ToList()};
+         var baseGrid = new BaseGrid(Constants.TIME, curveData.XAxis.Dimension) { Values = curveData.XValues.Select(x => x.X).ToList() };
 
          if (curveData.IsRange())
+         {
+            var rangeText = rangeDescriptions(curveData.Caption);
```

Comment:
you can have this guy return a tuple 
var (lowerRange, upperRange) = rangeDescriptuon(curveData.Caption);

That way there is no messinag around with [0] and [1]